src/types.ts

```diff
@@ -177,17 +177,17 @@ const IdentificationFlag = t.intersection([
 
 const CampaignFeatures = t.intersection([
   t.type({
-    minAppBinaryVersion: t.string,
-    minAppBuildVersion: t.number,
     campaignName: t.string,
-    transactionGrouping: t.boolean,
-    flowType: t.string,
     id: IdentificationFlag,
+    minAppBinaryVersion: t.string,
+    minAppBuildVersion: t.number,
   }),
   t.partial({
     alternateIds: t.array(IdentificationFlag),
     apiVersion: t.union([t.literal("v1"), t.literal("v2")]),
+    flowType: t.string,
     theme: t.union([t.literal("DEFAULT"), t.literal("GOVWALLET")]),
+    transactionGrouping: t.boolean,
```

Comment:
This is the only real change - moved `flowType` and `transactionGrouping` to "partial". The rest all ordering changes